financial-policy.md

```diff
@@ -0,0 +1,69 @@
+# denhac Financial Policy
+>[!NOTE]
+>Current Version Approved by The Board on December 6, 2022
+
+## How We Collect Money
+
+### Membership Dues:
+When a member registers for a denhac account, they will save a card on file with our card processor, Stripe. Under our Membership Agreement, a Membership is not considered active until an ID check has been performed. The member will be charged using the card on file with Stripe at the time of Access Card Activation. Membership dues are collected using Stripe on a monthly recurring basis. If we cannot charge the card on file for whatever reason, the member will be notified by Stripe and given four days to update their information. During this time, the member will still have access to the space, but after the failed attempt to charge their card, their membership will be terminated. The member can immediately update their form of payment after their membership termination and purchase a new membership.
+
+Current membership costs are posted in our Membership Agreement. All membership dues of current members will remain the same unless the impacted members are notified of this change within a reasonable amount of time approved by the Board.
+
+#### Cancellation Policy:
+At any time, a member can request to cancel their membership, effective at the start of their next renewal date. This request can be made through our website or by reaching out to the Treasurer. The Treasurer may approve other cancellation options on a case-by-case basis. 
+
+Any attempts to dispute a valid membership charge without a cancellation request will be treated as a cancellation request. The member will be prevented from obtaining a new membership unless the Treasurer approves.
+
+### Monetary Donations:
+From time to time, we will receive monetary donations from a member or third-party charity (Amazon Smile). All donations will be deposited into our general fund for space use unless otherwise directed by the donor. In this case, we will attempt to use the funds as directed and as circumstances permit.
```

Comment:
> (Amazon Smile)

LOL not anymore! 😸

> In this case, we will attempt to use the funds as directed and as circumstances permit.

Technically, and mostly just as a legal CYA, someone needs to approve this whether it's the board or treasurer. Otherwise you end up with someone donating $420 for a giant inflatable teddy bear. Unlikely, but possible. This is a great article on the subject: https://www.501c3.org/kb/what-are-restricted-funds/